### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by corranmac** [google-research/frame-interpolation#5]
Hi, I received this error
Traceback (most recent call last):
  File "/usr/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/content/frame_interpolation/eval/interpolator_test.py", line 94, in <module>
    app.run(main)
  File "/usr/local/lib/python3.7/dist-packages/absl/app.py", line 303, in run
    _run_main(main, args)
  File "/usr/local/lib/python3.7/dist-packages/absl/app.py", line 251, in _run_main
    sys.exit(main(argv))
  File "/content/frame_interpolation/eval/interpolator_test.py", line 90, in main
    _run_interpolator()
  File "/content/frame_interpolation/eval/interpolator_test.py", line 77, in _run_interpolator
    batch_dt)[0]
  File "/content/frame_interpolation/eval/interpolator.py", line 104, in interpolate
    result = self._model(inputs, training=False)
  File "/usr/local/lib/python3.7/dist-packages/tensorflow/python/saved_model/load.py", line 664, in _call_attribute
    return instance.__call__(*args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/tensorflow/python/eager/def_function.py", line 885, in __call__
    result = self._call(*args, **kwds)
  File "/usr/local/lib/python3.7/dist-packages/tensorflow/python/eager/def_function.py", line 957, in _call
    filtered_flat_args, self._concrete_stateful_fn.captured_inputs)  # pylint: disable=protected-access
  File "/usr/local/lib/python3.7/dist-packages/tensorflow/python/eager/function.py", line 1964, in _call_flat
    ctx, args, cancellation_manager=cancellation_manager))
  File "/usr/local/lib/python3.7/dist-packages/tensorflow/python/eager/function.py", line 596, in call
    ctx=ctx)
  File "/usr/local/lib/python3.7/dist-packages/tensorflow/python/eager/execute.py", line 60, in quick_execute
    inputs, attrs, num_outputs)
tensorflow.python.framework.errors_impl.InternalError:  dnn PoolForward launch failed
	 [[{{node StatefulPartitionedCall/StatefulPartitionedCall/average_pooling2d_1/PartitionedCall/AvgPool}}]] [Op:__inference_restored_function_body_13284]

Function call stack:
restored_function_body

**Comment by fitsumreda** [comment]
Does it run without errors if you use the images in ‘frame_interpolation/photos’?

You might need to set the _align parameter in eval/interpolator.py so it is evenly divisible to fit within film_net.

**Comment by corranmac** [comment]
It still fails using photos in frame_interpolation/photos but gives me this:
2022-02-15 01:48:51.560626: W tensorflow/core/common_runtime/gpu/gpu_bfc_allocator.cc:39] Overriding allow_growth setting because the TF_FORCE_GPU_ALLOW_GROWTH environment variable is set. Original config value was 0.
2022-02-15 01:49:02.859479: E tensorflow/stream_executor/cuda/cuda_dnn.cc:359] Loaded runtime CuDNN library: 8.0.5 but source was compiled with: 8.1.0.  CuDNN library needs to have matching major version and equal or higher minor version. If using a binary install, upgrade your CuDNN library.  If building from sources, make sure the library loaded at runtime is compatible with the version specified during compile configuration.
2022-02-15 01:49:02.860623: W ./tensorflow/stream_executor/stream.h:2109] attempting to perform DNN operation using StreamExecutor without DNN support
2022-02-15 01:49:02.866192: E tensorflow/stream_executor/cuda/cuda_dnn.cc:359] Loaded runtime CuDNN library: 8.0.5 but source was compiled with: 8.1.0.  CuDNN library needs to have matching major version and equal or higher minor version. If using a binary install, upgrade your CuDNN library.  If building from sources, make sure the library loaded at runtime is compatible with the version specified during compile configuration.
2022-02-15 01:49:02.867510: W tensorflow/core/framework/op_kernel.cc:1745] OP_REQUIRES failed at conv_ops.cc:1120 : UNKNOWN: Failed to get convolution algorithm. This is probably because cuDNN failed to initialize, so try looking to see if a warning log message was printed above.
2022-02-15 01:49:02.869623: E tensorflow/stream_executor/cuda/cuda_dnn.cc:359] Loaded runtime CuDNN library: 8.0.5 but source was compiled with: 8.1.0.  CuDNN library needs to have matching major version and equal or higher minor version. If using a binary install, upgrade your CuDNN library.  If building from sources, make sure the library loaded at runtime is compatible with the version specified during compile configuration.
2022-02-15 01:49:02.870628: W ./tensorflow/stream_executor/stream.h:2109] attempting to perform DNN operation using StreamExecutor without DNN support
2022-02-15 01:49:02.872168: E tensorflow/stream_executor/cuda/cuda_dnn.cc:359] Loaded runtime CuDNN library: 8.0.5 but source was compiled with: 8.1.0.  CuDNN library needs to have matching major version and equal or higher minor version. If using a binary install, upgrade your CuDNN library.  If building from sources, make sure the library loaded at runtime is compatible with the version specified during compile configuration.
2022-02-15 01:49:02.873220: W tensorflow/core/framework/op_kernel.cc:1745] OP_REQUIRES failed at conv_ops.cc:1120 : UNKNOWN: Failed to get convolution algorithm. This is probably because cuDNN failed to initialize, so try looking to see if a warning log message was printed above.
Traceback (most recent call last):
  File "/usr/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/content/frame_interpolation/eval/interpolator_test.py", line 94, in <module>
    app.run(main)
  File "/usr/local/lib/python3.7/dist-packages/absl/app.py", line 303, in run
    _run_main(main, args)
  File "/usr/local/lib/python3.7/dist-packages/absl/app.py", line 251, in _run_main
    sys.exit(main(argv))
  File "/content/frame_interpolation/eval/interpolator_test.py", line 90, in main
    _run_interpolator()
  File "/content/frame_interpolation/eval/interpolator_test.py", line 77, in _run_interpolator
    batch_dt)[0]
  File "/content/frame_interpolation/eval/interpolator.py", line 104, in interpolate
    result = self._model(inputs, training=False)
  File "/usr/local/lib/python3.7/dist-packages/tensorflow/python/saved_model/load.py", line 701, in _call_attribute
    return instance.__call__(*args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/tensorflow/python/util/traceback_utils.py", line 153, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "/usr/local/lib/python3.7/dist-packages/tensorflow/python/eager/execute.py", line 59, in quick_execute
    inputs, attrs, num_outputs)
tensorflow.python.framework.errors_impl.InternalError:  dnn PoolForward launch failed
	 [[node AvgPool
 (defined at /content/frame_interpolation/eval/interpolator.py:82)
]] [Op:__inference_restored_function_body_13284]

Errors may have originated from an input operation.
Input Source operations connected to node AvgPool:
In[0] inputs:

Operation defined at: (most recent call last)
>>>   File "/usr/lib/python3.7/runpy.py", line 193, in _run_module_as_main
>>>     "__main__", mod_spec)
>>> 
>>>   File "/usr/lib/python3.7/runpy.py", line 85, in _run_code
>>>     exec(code, run_globals)
>>> 
>>>   File "/content/frame_interpolation/eval/interpolator_test.py", line 94, in <module>
>>>     app.run(main)
>>> 
>>>   File "/usr/local/lib/python3.7/dist-packages/absl/app.py", line 303, in run
>>>     _run_main(main, args)
>>> 
>>>   File "/usr/local/lib/python3.7/dist-packages/absl/app.py", line 251, in _run_main
>>>     sys.exit(main(argv))
>>> 
>>>   File "/content/frame_interpolation/eval/interpolator_test.py", line 90, in main
>>>     _run_interpolator()
>>> 
>>>   File "/content/frame_interpolation/eval/interpolator_test.py", line 62, in _run_interpolator
>>>     model_wrapper = interpolator.Interpolator(_MODEL_PATH.value, _ALIGN.value)
>>> 
>>>   File "/content/frame_interpolation/eval/interpolator.py", line 82, in __init__
>>>     self._model = tf.compat.v2.saved_model.load(model_path)
>>> 

Function call stack:
restored_function_body -> average_pooling2d_1_layer_call_and_return_conditional_losses

I'm not sure how I would set the _align parameter?

**Comment by fitsumreda** [comment]
You need to fix this system environment error:
```
2022-02-15 01:49:02.859479: E tensorflow/stream_executor/cuda/cuda_dnn.cc:359] Loaded runtime CuDNN library: 8.0.5 but source was compiled with: 8.1.0. CuDNN library needs to have matching major version and equal or higher minor version. If using a binary install, upgrade your CuDNN library. If building from sources, make sure the library loaded at runtime is compatible with the version specified during compile configuration.
```

**Comment by fitsumreda** [comment]
Feel free to reopen if you have errors after setting up your system.

**Comment by kyrick** [comment]
I had these same errors in colab. I fixed it by `pip install apache-beam==2.34.0 mediapy==1.0.3` and **not installing requirements.txt.** Seems colab already has the necessary dependencies and requirements.txt was forcing a partial downgrade.
